### PR TITLE
Speed down with test

### PR DESCRIPTION
**Pull request by w1158mxs**
Decrease speed by clicking the arrow down, because users can increase the speed with the arrow up so they need to slow down without stopping. fix issue #51 - I face challenges in issue #48 so I redo it please ignore 48

### CODE REVIEW

**Review by bakursait2 — CHANGES_REQUESTED** [review]
It is a cool feature to increase/decrease the speed. However, would not be better if we change the speed keys from arrowUp and arrowDown to something that is more handy to the user?

I mean it might be hard for a player to leave the mouse just to hit increase(arrowUp)/decrease(arrowDown) bottons. Instead we can use some letters that is close to his fingers such as (for example) `q` for speedUp and `e` for speedDown.
or we can use the mouse wheel to increase/decrease the speed.

**Review by xieWright — APPROVED** [review]
Passed all the tests in the auto testing and the code looks fine to me. Also adding self to the pyproject.toml works.

**Review by bakursait2 — APPROVED** [review]

**Review by Lurk390 — APPROVED** [review]
I agree with Abu Bakur, speed-up and speed-down would be better as different keys to improve user experience, however, this would be future feature to implement. Good work.

File: tempus_fugit_minecraft/main.py [hunk 1]
```diff
@@ -859,10 +859,15 @@ def on_key_press(self, symbol, modifiers):
             self.block = self.inventory[index]
         elif symbol == key.UP:
             self.speed_up()
+        elif symbol == key.DOWN:
+            self.speed_down()
```
[the remark this answers is not in the document]
Comment (w1158mxs) [reply tempus_fugit_minecraft/main.py:877]: I agree with your idea to make one hand on the mouse and the other on the keyboard, I will make it a new issue.

**Review by criticalglitch — APPROVED** [review]

**Review by w1158mxs — COMMENTED** [review]
Test should be changes because we did merge two files to one test file

**Comment by w1158mxs** [comment]
Test passed
speed_down_with_test_screenshot
for others, all tests are automated

**Pull request merged**